blake3/blake3_test.go

```diff
@@ -0,0 +1,38 @@
+// Copyright 2021 OCI Contributors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+package blake3
+
+import (
+	"testing"
+
+	"github.com/opencontainers/go-digest/testdigest"
+	_ "github.com/zeebo/blake3"
```

Comment:
This import does not seem needed

Reply:
Fixed

Reply:
Doesn't seem so https://github.com/opencontainers/go-digest/blob/0f543f5712a3617a5918db97ce7cd8a789df129b/blake3/blake3_test.go#L20

Reply:
Huh. No idea what happened. Re-force-pushed, and it seems to have updated this time.